src/main/java/com/google/gcs/sdrs/SdrsApplication.java

```diff
@@ -165,9 +175,17 @@ private static String getPropertyValueFromEnv(String token) {
     return System.getenv(envVariable);
   }
 
-  private static void registerPubSub() {
+  private static boolean registerPubSub() {
```

Comment:
For the change to a  boolean return type from void consider changing the method name to "is" as is the convention for conveying clear readability

Reply:
change it to void and log info inside of the method